server/src/main/java/org/jboss/as/server/parsing/SystemPropertiesXml.java

```diff
@@ -130,6 +133,18 @@ void parseSystemProperties(final XMLExtendedStreamReader reader, final ModelNode
             if (!setName) {
                 throw ParseUtils.missingRequired(reader, Collections.singleton(NAME));
             }
+
+            Properties properties = WildFlySecurityManager.getSystemPropertiesPrivileged();
+            try {
+                String newPropertyValue = SystemPropertyResourceDefinition.VALUE.resolveValue(ExpressionResolver.SIMPLE, op.get(VALUE)).asString();
+                String oldPropertyValue = properties.getProperty(name);
+                if (oldPropertyValue != null && !oldPropertyValue.equals(newPropertyValue)) {
+                    ControllerLogger.ROOT_LOGGER.systemPropertyAlreadyExist(name, oldPropertyValue, newPropertyValue);
+                }
+            } catch (OperationFailedException e) {
+                ServerLogger.AS_ROOT_LOGGER.tracef(e, "Failed to set property %s at parse time, it will be set later in the boot process", name);
```

Comment:
is the message correct? You are not setting the property here, you just check its resolved value against the `System.getProperties()`.

Reply:
Thanks. This message is copied from trace message a few lines below, I will adjust it.